src/PrestaShopBundle/Form/Admin/Sell/Address/ManufacturerAddressType.php

```diff
@@ -244,24 +265,30 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ]),
                 ],
             ])
-            ->add('home_phone', TextType::class, [
+            ->add('dni', TextType::class, [
+                'label' => $this->trans('DNI', 'Admin.Global'),
                 'required' => false,
                 'empty_data' => '',
                 'constraints' => [
+                    new AddressDniRequired([
+                        'required' => false,
+                        'id_country' => $countryId,
+                    ]),
                     new TypedRegex([
-                        'type' => 'phone_number',
+                        'type' => 'dni_lite',
                     ]),
                     new Length([
-                        'max' => AddressSettings::MAX_PHONE_LENGTH,
-                        'maxMessage' => $this->translator->trans(
+                        'max' => 16,
```

Comment:
It isn't changed, but because I moved some fields around in form git diff is a mess now.

Notice the rest of the changes to this type, it has been changed to dni field (and original dni has been changed to mobile phone and mobile phone has bee changed to home phone).

Reply:
You're right! My bad 😅